doc/barclamp.md

```diff
@@ -1356,3 +1356,147 @@ place) you will have to add additional setup code to
 and/or
 [qa_crowbarsetup.sh](https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crobarsetup.sh)
 to take care of your barclamp's needs.
+
+## High Availability (HA) Integration
+
+Using the [barbican barclamp](https://github.com/crowbar/crowbar-openstack/tree/9805c7ddb81ffbf6f2b79061c6ceb332659cd614/chef/cookbooks/barbican)
+as an example. We will try to investigate what changes are required to make a barclamp
+HA compatible
+
+[This commit](https://github.com/crowbar/crowbar-openstack/pull/512/files) is
+the first place we start looking. It was not entirely correct and needed
+[more commits](https://github.com/crowbar/crowbar-openstack/commits/master/chef/cookbooks/barbican)
+to correct.
+
+-  [default.rb](https://github.com/crowbar/crowbar-openstack/commits/master/chef/cookbooks/barbican/attributes/default.rb)
+
+   This file defines the default for all the values of the vars required by barclamp.
```

Comment:
Don't indent the beginning of normal text paragraphs, since indentation has specific meaning in Markdown.